<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>DepartmentEditView</class>
<widget class="QWidget" name="DepartmentEditView">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>500</width>
   <height>500</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>科室信息编辑</string>
 </property>
 <layout class="QVBoxLayout" name="verticalLayout">
  <item>
   <widget class="QGroupBox" name="groupBox">
    <property name="title">
     <string>基本信息</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <!-- 新增：科室ID显示 -->
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>科室ID：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtID">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>自动生成</string>
       </property>
      </widget>
     </item>
     
     <!-- 原科室名称 -->
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>科室名称：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtName">
       <property name="placeholderText">
        <string>请输入科室名称</string>
       </property>
      </widget>
     </item>
     
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>位置/楼层：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtLocation">
       <property name="placeholderText">
        <string>如：门诊大楼3楼</string>
       </property>
      </widget>
     </item>
     
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>科室电话：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtPhone">
       <property name="placeholderText">
        <string>如：028-12345678</string>
       </property>
      </widget>
     </item>
     
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>成立日期：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateEstablished">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>床位数量：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBedCount">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="suffix">
        <string> 张</string>
       </property>
      </widget>
     </item>
     
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>科室主任：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="txtDirector">
       <property name="placeholderText">
        <string>请输入医生ID或姓名</string>
       </property>
      </widget>
     </item>
     
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>状态：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="cmbStatus">
      </widget>
     </item>
    </layout>
   </widget>
  </item>
  
  <item>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="title">
     <string>科室描述</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="txtDescription">
       <property name="placeholderText">
        <string>请输入科室描述信息...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </item>
  
  <!-- 错误信息显示 -->
  <item>
   <widget class="QLabel" name="lblError">
    <property name="text">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </item>
  
  <!-- 按钮区域 -->
  <item>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>